README.md

```diff
@@ -1,2 +1,41 @@
-# proposals
-This repository holds all the past and current proposals for the Mixxx Software
+# Proposals
+
+This repository holds all the past and current proposals for the Mixxx Software. It’s the single place for
+reviewing, discovering, and working on the design documents. It is also a record of past decisions and approvals.
+
+## Current Proposals
+
+* The [open PRs with the `proposal` label](https://github.com/mixxxdj/proposals/pulls?q=is%3Aopen+is%3Apr+label%3Aproposal) show all the pending proposals.
+* The [proposals directory](./proposals) shows all the accepted proposals. See the “Implementation Status” for details on the implementation.
+* The [PRs with the `proposal` label that are closed without merging](https://github.com/mixxxdj/proposals/pulls?q=is%3Apr+label%3Aproposal+is%3Aclosed+is%3Aunmerged) show all the rejected proposals.
+
+## What’s a Design Document?
+
+It’s essential to clearly explain the reasons behind certain design decisions to have a community consensus. This is especially
+important in Mixxx, where every decision might have a significant impact given the high adoption and stability of the software and standards we work on.
+
+In our world, no decision is perfect, so having a design document explaining our trade-offs is essential.
+Such a document can also be used later as a reference and for knowledge-sharing purposes.
+
+Design documents do not always reflect what has been (or will be) implemented. Implementation details
+might have changed since a feature was merged. Design docs are not considered documentation and can not define a standard.
+Instead, it should explain the motivation, scope, decisions, and alternatives considered.
```

Comment:
A design document shall describe the use-cases the proposal will implement and also the use-cases that are explicitly not included. 

It may also contain requirements and acceptance criteria of a derived Pull Request. This avoid feature creep and adjust the expectations during a review.